baseband/vlbi_base/base.py

```diff
@@ -345,7 +350,7 @@ def seek(self, offset, whence=0):
                 whence = 0
 
             offset = offset.to(u_sample, equivalencies=[(u.s, u.Unit(
-                self.samples_per_frame * self.frames_per_second * u_sample))])
+                self._sample_rate * u_sample))])
```

Comment:
Here, like for `tell`, I think on second sight, one can simplify hugely and just do
```
offset = int(round(offset * self.sample_rate))
```

That probably means we do not need `u_sample` any more...

Reply:
As above.